packages/sdk/app-framework/src/plugins/common/navigation.ts

```diff
@@ -156,6 +156,7 @@ export enum NavigationAction {
   SET = `${NAVIGATION_ACTION}/set`,
   ADJUST = `${NAVIGATION_ACTION}/adjust`,
   CLOSE = `${NAVIGATION_ACTION}/close`,
+  EXPOSE = `${NAVIGATION_ACTION}/expose`,
```

Comment:
I'm skeptical about this being a navigation action, but we already have a bunch of navigation actions that are pretty specific to deck so not too worried about it at this stage